src/Operation/InsertOne.php

```diff
@@ -21,6 +21,8 @@
 use MongoDB\Driver\BulkWrite as Bulk;
 use MongoDB\Driver\Server;
 use MongoDB\Driver\WriteConcern;
+use MongoDB\Driver\Exception\BulkWriteException as BulkWriteException;
```

Comment:
There is no need to use "as BulkWriteException" here, as the aliased named is the same as the original name - just like you have done for "MongoDB\Exception\DuplicateKeyException;" the line below.

Reply:
Thank you!